core/trashcan.js

```diff
@@ -166,8 +166,9 @@ Blockly.Trashcan.prototype.createDom = function() {
   */
   this.svgGroup_ = Blockly.utils.createSvgElement('g',
       {'class': 'blocklyTrash'}, null);
+  var clip;
   var rnd = String(Math.random()).substring(2);
-  var clip = Blockly.utils.createSvgElement('clipPath',
```

Comment:
In the past it has been Google style to redeclare variables when they are used again for something else.  This makes code easier to maintain.  But now the Closure Compiler is screaming about it.  Fine, whatever.